proto/plan.proto

```diff
@@ -254,6 +254,7 @@ message ColDef {
 	bool primary        = 5;
 	int32 pkidx 		= 6;
 	string comment	=7;
+	bool isHidden = 8;
```

Comment:
use "is_hidden" in proto

Reply:
isHiddenKey is used for no primary key situation, I think if i use this key, i need to compatible with no primary key situation, I think we have three diff situation : single primary key, is hidden key, multi primary key.  there can't be mixed.